packages/kbn-ui-framework/Gruntfile.js

```diff
@@ -150,6 +151,7 @@ module.exports = function(grunt) {
       sass.render(
         {
           file: src,
+          fiber: Fiber,
         },
```

Comment:
I made this optional change to boost performance:

> Note however that by default, **`renderSync()` is more than twice as fast as `render()`** due to the overhead of asynchronous callbacks. To avoid this performance hit, `render()` can use the [`fibers`](https://www.npmjs.com/package/fibers) package to call asynchronous importers from the synchronous code path.

Reference: https://github.com/sass/dart-sass#javascript-api